### PR TITLE
refactor(ci/pr-test): extract global BUILD_OUT_ROOT env

### PR DESCRIPTION
**Pull request by yin1999**
### Description

simplify the "PR test" workflow

### Related issues and pull requests

Port: mdn/content#26170

### CODE REVIEW

**Comment by yin1999** [comment]
Tested in #15169: https://github.com/mdn/translated-content/actions/runs/5887250360/job/15966335645?pr=15169#step:7:35

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -23,6 +23,10 @@ jobs:
     env:
       BASE_SHA: ${{ github.event.pull_request.base.sha }}
       HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+      # If we don't do this the built files will end up in
+      # `node_modules/@mdn/yari/client/build/` and we don't want that
+      # to get pushed into the cache.
+      BUILD_OUT_ROOT: /tmp/build
```
Comment (yin1999) [inline .github/workflows/pr-test.yml:29]: This should also work, see [GitHub docs](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow)

**Review by caugner — APPROVED** [review]
LGTM.

PS: `refactor(ci/pr-test): extract global BUILD_OUT_ROOT env` could be a more specific title.

**Comment by yin1999** [comment]
> LGTM.
> 
> PS: `refactor(ci/pr-test): extract global BUILD_OUT_ROOT env` could be a more specific title.

Thanks for the review @caugner. I've updated the title :)

**Pull request merged**